### ISSUE TITLE
Firefox extension version is outdated

### ISSUE THREAD

**Issue by zumoshi** [pontem-network/pontem-wallet#14]
**Describe the bug**
The official links from [pontem.network](https://pontem.network/pontem-wallet) link to [chrome store](https://chromewebstore.google.com/detail/pontem-aptos-wallet/phkbamefinggmakgklpkljjmgibohnba) which has version Version
2.6.0 Updated February 12, 2024, and [firefox addon](https://addons.mozilla.org/en-US/firefox/addon/pontem-aptos-wallet/) which is Version 2.3.3 Last updated Jan 24, 2023.

Many already fixed bugs still exist in the old version.

**To Reproduce**
1. Go to https://pontem.network/pontem-wallet
2. Click on Firefox extension
3. Install the addon
4. Create a wallet
5. Choose test network and click on Faucet
6. You will be redirected to the outdated page on aptos site: https://aptoslabs.com/testnet-faucet
(This is an example, there are many other issues that are not present in the newer version on chrome)

**Expected behavior**
To get test coins from test net faucet.

**Screenshots**
Firefox vs chrome:
![image](https://github.com/pontem-network/pontem-wallet/assets/4184939/f46c5d44-d0d2-4677-a369-f232e3907adc)
![image](https://github.com/pontem-network/pontem-wallet/assets/4184939/4b822d18-0f87-4f43-9a8f-43ea5b4ce6be)
![image](https://github.com/pontem-network/pontem-wallet/assets/4184939/573cb47a-ddbc-4068-b71b-cd1a25403022)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser Firefox 123.0.1 (64-bit)
 - Version 2.3.3

**Smartphone (please complete the following information):**
N/A

**Additional context**
Please push the latest release to the Firefox addon store.
Thanks.

**Comment by bjeschke** [comment]
Yes, same here. My firefox wallet is broken. You should provde a message or something. Now I cannot use the wallet anymore.

<img width="406" alt="grafik" src="https://github.com/pontem-network/pontem-wallet/assets/14287880/5222d6a2-2927-4cd7-b7b6-674db2b5a0cf">

**Comment by Sanjasin** [comment]
+++
